tensorflow/contrib/layers/python/layers/layers.py

```diff
@@ -1445,7 +1445,8 @@ def fully_connected(inputs,
     ValueError: If x has rank less than 2 or if its last dimension is not set.
   """
   if not isinstance(num_outputs, six.integer_types):
-    raise ValueError('num_outputs should be int or long, got %s.', num_outputs)
+    raise ValueError(
+        'num_outputs should be int or long, got %s.' % (num_outputs,))
```

Comment:
What if num_outputs is a tuple? Then you'd get an obscure TypeError: not
all arguments converted during string formatting. This ValueError is only
raised if the type is wrong in the first place (as I experienced), so in my
opinion it's helpful to be robust to this case.

What do you think?